### PR TITLE
Added Wise Banking to Connectivity Apps

### PR DESCRIPTION
**Pull request by dagnybjork**
Added Wise Banking to the connectivity apps feature. Created a new procedure 'RegisterWiseBanking' and call to that procedure from LoadBankingAppsData.

### CODE REVIEW

**Comment by dagnybjork** [comment]
> @dagnybjork please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement

@microsoft-github-policy-service agree [company="Wise lausnir hf."]

**Comment by dagnybjork** [comment]
@microsoft-github-policy-service agree company="Wise lausnir hf."

File: Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al [hunk 1]
```diff
@@ -82,6 +83,34 @@ codeunit 20352 "Connectivity App Definitions"
...
+    begin
+        /***************************************************
+            Add app 'Wise Banking' to IS
+        ***************************************************/
+
+        AppId := '6a580fed-3f05-40d3-bd61-652c7af0622f';
+        AppName := 'Wise Banking';
+        AppPublisher := 'Wise';
+        AppDescription := 'Wise Banking provides the user with secure communication with all commercial banks of Iceland. It does this by using the banking standards of the Icelandic banks (IOBS). With Wise Banking the user can therefore safely manage bank accounts, automatic bank reconciliation, outgoing payments and currency and exchange rates directly from Business Central.';
+        AppProviderSupportURL := 'https://wise.is/en/solutions/wisebusiness/wise-banking/';
```
Comment (sorenfriisalexandersen) [inline Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al:105]: Great description. 
As for the AppProviderSupportURL, make sure you will display supported bank names directly on that landing page, so it's immediately visible when the user navigates to this page from BC. (I think you are already working on this. All good).

**Review by sorenfriisalexandersen — DISMISSED** [review]
Only added a comment. Everything else looks good from my perspective.

**Comment by haoranpb** [comment]
It's a minor thing, but I'd move the code to line 170, so it's easier to tell the order of when an app is added

File: Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al (hunk 1 above)
Comment (dagnybjork) [reply Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al:105]: We have added a list of supported banks to the site.

**Comment by dagnybjork** [comment]
> It's a minor thing, but I'd move the code to line 170, so it's easier to tell the order of when an app is added

I have moved the code to line 170.

**Review by haoranpb — DISMISSED** [review]

**Review by darjoo — DISMISSED** [review]
@haoranpb there's a conflict from the other PR now

**Review by haoranpb — APPROVED** [review]

**Comment by haoranpb** [comment]
merged internally, just need one more vote

**Review by JesperSchulz — APPROVED** [review]

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 51012.

**Comment by JesperSchulz** [comment]
Availability update: We will publish a fix for this issue in the next update for release 21.

Build ID to track: 51135.

**Pull request merged**